get-started/start-building/batch.md

```diff
@@ -317,18 +397,25 @@ Use the following Python script or curl command examples to list all of your bat
 
 To cancel a batch job currently in progress, send a request to the `cancel` endpoint with your `batch_id`. Note that cancellation may take up to 10 minutes to complete, when the status will show as `cancelling`. Once complete, the status will show as `cancelled`.
 
-Use the following Python script or curl command examples to cancel a batch job:
+You can use the following snippets to cancel a batch job:
 
 === "Python"
 
     ```python title="Example"
     from openai import OpenAI
-
+    from getpass import getpass
+    
+    # Get API key from user input
+    api_key = getpass("Enter your kluster.ai API key: ")
```

Comment:
Should probably use the usual syntax here ("INSERT_API_KEY") and move this to a code snippet